ruma-api/tests/conversions.rs

```diff
@@ -1,4 +1,6 @@
-use ruma_api::{ruma_api, IncomingRequest as _, OutgoingRequest as _};
+use ruma_api::{
+    ruma_api, IncomingRequest as _, OutgoingRequest as _, OutgoingRequestAppserviceExt as _,
```

Comment:
Why are these imported here like this?

(not a criticism, genuinely curious why side-effect imports like this are needed)

Reply:
Without importing traits the trait methods aren't available in the scope

Reply:
If the question was regarding why `as _` is used, then its mostly a matter of taste I guess, as long as there's no name clash it doesn't really matter

Reply:
Ahhh, gotcha.

This thread can be closed then, I wasn't aware you couldn't use trait methods without importing the trait itself too.